Kickstarter-iOS/Views/Cells/PledgeShippingLocationCell.swift

```diff
@@ -53,6 +53,8 @@ final class PledgeShippingLocationCell: UITableViewCell, ValueCell {
     _ = self.adaptableStackView
       |> checkoutAdaptableStackViewStyle(self.traitCollection.ksr_isAccessibilityCategory())
 
+    _ = self.amountLabel
+      |> checkoutBackgroundStyle
```

Comment:
Setting background color on some of the labels to get rid of blended layers (in spirit of performance improvement and best practices)

| Before 🐛 | After 🦋 |
| --- | --- |
| <img width="496" alt="Screen Shot 2019-05-01 at 12 09 16 PM" src="https://user-images.githubusercontent.com/387596/57027502-88141100-6c0a-11e9-9275-6f7a9f673245.png"> | <img width="515" alt="Screen Shot 2019-05-01 at 12 07 28 PM" src="https://user-images.githubusercontent.com/387596/57027507-8b0f0180-6c0a-11e9-9f49-c44854dba128.png"> |